Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ASM_Alert.yml

```diff
@@ -2621,34 +2767,40 @@ tasks:
     quietmode: 0
     isoversize: false
     isautoswitchedtoquietmode: false
-  "112":
-    id: "112"
-    taskid: 710aec66-336d-4cbc-8cbd-0fc5198f5e4c
+  "125":
+    id: "125"
+    taskid: 17079024-16ff-45ac-813a-f89e32f4cd6b
     type: regular
     task:
-      id: 710aec66-336d-4cbc-8cbd-0fc5198f5e4c
+      id: 17079024-16ff-45ac-813a-f89e32f4cd6b
       version: -1
-      name: Close alert (remediated automatically)
-      description: Close the current alert because Automated Remediation path was selected.
-      script: Builtin|||closeInvestigation
+      name: Set playbook stage grid field (enrichment)
+      description: |-
+        Automation used to more easily populate a grid field.  This is necessary when you want to assign certain values as static or if you have context paths that you will assign to different values as well.  Instead of a value you can enter `TIMESTAMP` to get the current timestamp in ISO format.  Example of command:
```

Comment:
```suggestion
        Automation used to more easily populate a grid field. This is necessary when you want to assign certain values as static or if you have context paths that you will assign to different values as well. Instead of a value, you can enter `TIMESTAMP` to get the current timestamp in ISO format.  Example of the command:
```